insights/client/__init__.py

```diff
@@ -523,4 +523,10 @@ def collect():
 
 def upload():
     egg_path = sys.stdin.read().strip()
-    run("upload", egg_path)
+    if not config["to_json"]:
```

Comment:
can we invert the `if` to avoid negative boolean logic?

Reply:
Can do coach.